internal/backend/db/dbgorm/connections_test.go

```diff
@@ -46,11 +46,11 @@ func TestCreateConnectionForeignKeys(t *testing.T) {
 
 	// FIXME: ENG-571
 	// c.IntegrationID = &unexisting
```

Comment:
Delete commented code.

Reply:
it's under FIXME and should be re-enabled afer fixing this.

Reply:
Some wording after the issue number would help.

Reply:
done

Reply:
done